src/pocketmine/entity/Entity.php

```diff
@@ -1634,7 +1634,7 @@ protected function checkGroundState(float $movX, float $movY, float $movZ, float
 	 */
 	public function getBlocksAround() : array{
 		if($this->blocksAround === null){
-			$bb = $this->boundingBox->grow(0.01, 0.01, 0.01);
+			$bb = clone $this->boundingBox;
```

Comment:
why clone when it's only being read? 🤔

Reply:
it was a quickfix because I didn't have time to go through and change all the `bb` to `this->boundingBox`. Obviously I wasn't thinking very clearly.

Reply:
Additionally this should actually be shrink() because floating-point errors can result in you "intersecting" with a block that you're only next to. As I recall the grow() was a change for magma blocks to work, but it causes lots of undesired behaviour.